modules/sql/src/main/resources/db/migration/V001__Initial_Setup.sql

```diff
@@ -61,7 +61,7 @@ ALTER TYPE f2_decker OWNER TO postgres;
 --
 
 CREATE DOMAIN identifier AS character varying(32)
-	CONSTRAINT identifier_check CHECK (((VALUE)::text ~ '^[A-Z][A-Za-z0-9]*$'::text));
+	CONSTRAINT identifier_check CHECK (((VALUE)::text ~ '^[A-Z][A-Za-z0-9_]*$'::text));
```

Comment:
It's the name of any enumeration element like a GMOS disperser:

```
CREATE TABLE e_gmos_north_disperser (
    id             identifier            PRIMARY KEY,
...
);
```

The check verifies that these names follow the given regex.  Since GMOS filter options combine various filters like `ZPrime` and `CaT` into `ZPrime_CaT`, I added support for underscore.